c++/src/capnp/rpc-twoparty.h

```diff
@@ -105,8 +105,22 @@ class TwoPartyVatNetwork: public TwoPartyVatNetworkBase,
   kj::Promise<kj::Own<TwoPartyVatNetworkBase::Connection>> accept() override;
 
 private:
-  class OutgoingMessageImpl;
   class IncomingMessageImpl;
+  class OutgoingMessageImpl final
```

Comment:
You don't need to move this class definition to the header if you instead declare `TwoPartyVatNetwork`'s destructor out-of-line.